app/models/enterprise.rb

```diff
@@ -338,6 +340,22 @@ def can_invoice?
     abn.present?
   end
 
+  def delete_logo
+    @delete_logo ||= '0'
+  end
+
+  def delete_logo=(value)
+    @delete_logo = value
+  end
+
+  def delete_promo_image
+    @delete_promo_image ||= '0'
+  end
+
+  def delete_promo_image=(value)
+    @delete_promo_image = value
+  end
```

Comment:
`attr_accessor` already provides getters and setters; I think that the default value of `'0'` is not needed, sine `nil` is falsey as well. Less code, and less to test.